.idea/compiler.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="1.8" />
+    <bytecodeTargetLevel target="11" />
```

Comment:
Java 11 is required for 7.0+ Android Gradle plugin. I tested building an AAR and importing it into a Java 8 project, and there were no issues. The declared Java version in each module's build.gradle file is still Java 8.